packages/driver/src/cypress.ts

```diff
@@ -415,6 +431,8 @@ class $Cypress {
         // test:after:run events ourselves
         this.emit('run:end')
 
+        this.maybeEmitCypressInCypress('mocha', 'end', args[0])
```

Comment:
Needed to capture the inner Cypress events in the outer Cypress. Not super ideal, I wonder if there's a cleaner way to do this. I tried spying on the inner Mocha but I could not get this to work.